docs/src/PolyhedralGeometry/fans.md

```diff
@@ -71,6 +71,7 @@ lineality_dim(PF::PolyhedralFan)
 lineality_space(PF::PolyhedralFan)
 maximal_cones(PF::PolyhedralFan)
 maximal_cones_as_incidence_matrix(PF::PolyhedralFan)
```

Comment:
Can we get rid of `maximal_cones_as_incidence_matrix` and merge this with `maximal_cones`? Via the `as` parameter?